detect-nuget-inspector/detect-nuget-inspector/Inspection/Inspectors/SolutionInspector.cs

```diff
@@ -145,7 +145,10 @@ public Container GetContainer()
                                 {
                                     foreach (Container container in projectResult.Containers)
                                     {
-                                        container.Dependencies.AddRange(packages);
+                                        if (container != null && container.Dependencies != null)
```

Comment:
Is there a way to instead ensure that there isn't even a null reference in the list to begin with? If so, would that be preferable?